api/Element.json

```
@@ -7427,13 +7427,16 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/Element/scrollLeft",
           "support": {
             "chrome": {
-              "version_added": "43"
+              "version_added": "43",
+              "notes": "Uses 0-100 for RTL instead of negative values. See <a href='https://crbug.com/721759'>issue 721759</a>"
             },
             "chrome_android": {
-              "version_added": "43"
+              "version_added": "43",
+              "notes": "Uses 0-100 for RTL instead of negative values. See <a href='https://crbug.com/721759'>issue 721759</a>"
             },
             "edge": {
-              "version_added": "12"
+              "version_added": "12",
+              "notes": "Uses 0-100 for RTL instead of negative values. See <a href='https://crbug.com/721759'>issue 721759</a>"
```

Based on the discussion in the linked bug, it sounds like this behavior changed a bit between EdgeHTML Edge and Chromium Edge. I'd like to suggest representing this as two notes in an array:

(I think I have the source formatting right here, but there may be CI errors if I'm wrong)

```suggestion
              "notes": [
                "From Edge 79, for right-to-left elements, this property uses 0-100 (most left to most right) instead of negative values. See <a href='https://crbug.com/721759'>bug 721759</a>.",
                "Before Edge 79, for right-to-left elements, this property uses 100-0 (most left to most right) instead of negative values."
              ]
```